frontend/src/pages/item/_itemId/index.vue

```diff
@@ -316,7 +316,7 @@ const item = (
 const currentSource = ref<MediaSourceInfo>({});
 const currentVideoTrack = ref<number>();
 const currentAudioTrack = ref<number>();
-const currentSubtitleTrack = ref<number>();
+const currentSubtitleTrack = ref<number | undefined>();
```

Comment:
```suggestion
const currentSubtitleTrack = ref<number>();
```

Same as above, I think this is already resolved by TS as ``number | undefined`` without the explicit undefined.